lib/cli-engine.js

```diff
@@ -256,6 +256,27 @@ function isErrorMessage(message) {
     return message.severity === 2;
 }
 
+
+/**
+ * Checks if a provided path is a directory and returns a glob string matching
+ * all files under that directory if so, the path itself otherwise.
+ * Also makes sure all path separators are POSIX style for `glob` compatibility.
+ *
+ * @param {string} pathname The directory path to be modified
+ * @returns {string} The glob path or the file path itself
+ */
+function processPath(pathname) {
```

Comment:
Will be good to have in the comment. Since whoever reads this after 1 year can understand. there is no limit on comments. :smile: